core-bundle/contao/languages/en/modules.xlf

```diff
@@ -311,6 +311,12 @@
       <trans-unit id="FMD.rssReader.1">
         <source>Adds an RSS feed to the page</source>
       </trans-unit>
+      <trans-unit id="FMD.feed_reader.0">
```

Comment:
So basically this is "just" a new FE module now, replacing the old one, right?
We should adjust the title of the existing one to `RSS reader (deprecated)` :)

Reply:
I labeled the RSS reader module as deprecated.